src/around.js

```diff
@@ -0,0 +1,34 @@
+var curryN = require('./curryN');
+/**
+ * Accepts three functions, `fore`, `hind` and `fn`, and a single value.
+ * `R.around` effectively wraps `fn` with `fore` and `hind` by first passing the
+ * given value to `fore`, which returns a value that is then passed to `fn`,
+ * whose return value is finally passed to `hind`. The return value of
+ * `R.around` is the result of this call to `hind`.
+ *
+ * This can be useful for converting some type `a` to be compatible with some
+ * function `(b -> b)` and back again, where an isomorphism exists between
+ * `a` and `b`: `R.around(aToB, bToA, bFn)`.
+ *
+ * @func
+ * @memberOf R
+ * @category Function
+ * @sig (a -> b) -> (c -> d) -> (b -> c) -> a -> d
+ * @param {Function} fore
+ * @param {Function} hind
+ * @param {Function} fn
+ * @param {*}        x
+ * @return {*}
+ * @example
+ *
+ *      var toCharCodes = R.compose(R.map(R.invoker(1, 'charCodeAt')(0)), R.split(''));
+ *      var fromCharCodes = R.apply(String.fromCharCode);
+ *      var strToCharCodes = R.around(toCharCodes, fromCharCodes);
+ *      var printableCharCodes = R.reject(R.either(R.lt(R.__, 32), R.equals(127)));
+ *      var printableStr = strToCharCodes(printableCharCodes);
+ *
+ *      printableStr("foo\nbar\tbaz"); //=> "foobarbaz"
```

Comment:
I agree that the name might be misleading. When I first read the title of the PR I thought it might have to do with rounding floats.

I propose the name `R.sandwich(f, g, h)`, since it's kind of like making a function sandwich where the first 2 functions are the bread and the last function is the delicious filling.